### PR TITLE
✨Lentainform new improvements

### PR DESCRIPTION
**Pull request by DenyHF**
Hello!
We implemented a couple of amp features into our code!
thanks!

### CODE REVIEW

**Comment by torch2424** [comment]
@DenyHF Will take a look on Monday 😄

**Comment by torch2424** [comment]
Unfortunately, got pulled onto a super high priority project. 😢 Can you take this @zhouyx ?

**Comment by DenyHF** [comment]
Hi, thanks for the feedback.
Waiting for your decision.

**Comment by zhouyx** [comment]
Hello @DenyHF, thank you for summiting the change. I found the change very similar to #22907. I figure it would be easier to work on that PR first. I'll review this PR once we have that one merged. Thanks.

File: 3p/integration.js [hunk 1]
```diff
@@ -278,6 +278,7 @@ const AMP_EMBED_ALLOWED = {
   'mantis-recommend': true,
   miximedia: true,
   mywidget: true,
+  lentainform: true,
```
Comment (zhouyx) [inline 3p/integration.js]: nit: sort alphabetically please.

Comment (zhouyx) [reply 3p/integration.js]: let's also fix the order please : )

**Comment by zhouyx** [comment]
PR merged. Thank you

**Pull request merged**